bestla/bestla/kernel_jit.h

```diff
@@ -252,6 +252,123 @@ class DequanS8FP {
 };
 class DecompresssS3 {
```

Comment:
should be named DecompressS3 ... my bad....

Reply:
fixed